scheduler/api/server.go

```diff
@@ -78,3 +78,25 @@ func (h *Handler) GetSchedulerStatus() (*scoot.SchedulerStatus, error) {
 func (h *Handler) SetSchedulerStatus(maxNumTasks int32) error {
 	return schedthrift.SetSchedulerStatus(h.scheduler, maxNumTasks)
 }
+
+// Implements GetClassLoadPcts Cloud Scoot API
+func (h *Handler) GetClassLoadPcts() (map[string]int32, error) {
+	return schedthrift.GetClassLoadPcts(h.scheduler), nil
+}
+
+// Implements SetClassLoadPcts Cloud Scoot API
+func (h *Handler) SetClassLoadPcts(classLoadPcts map[string]int32) error {
+	schedthrift.SetClassLoadPcts(h.scheduler, classLoadPcts)
+	return nil
```

Comment:
why not just return an err from SetClassLoadPcts and return that here?

Reply:
adding